### DISCUSSION TITLE
Withdraw

### DISCUSSION THREAD

**Discussion by murselkeskin5858** [IBAX-io/go-ibax#567] [Rewards]
When will i withdraw my airdop?

**Comment by hes91** [comment]
1 şubat itibariyle size verilecek olan IBXC lerin %10 luk kısmını çekebileceksiniz.Hepsini çekebilmen için 6 ay geçmesi gerekli.

**Reply by ahmetce37** [reply]
6 ay süreye yayılması güzel. Fiyatın korunması açısından mantıklı olmuş.

**Comment by ibaxio** [comment]
Once the mainnet of IBAX is launch, it will be distributed based on the cleared data of airdrop.
